src/components/Menu/Menu/Menu.tsx

```diff
@@ -104,6 +90,39 @@ const Menu: VibeComponent<MenuProps> & {
       });
     }, [originalChildren]);
 
+    const updateActiveItemIndex = useCallback(
+      (index: number) => {
+        setActiveItemIndex(index);
+
+        const activeChild = children[index];
+        const ariaActiveDescendant = React.isValidElement(activeChild)
+          ? activeChild?.props?.id || `${id}-${index}`
+          : undefined;
+        if (ariaActiveDescendant) {
```

Comment:
As we talked before: let's add console warning if developer did not define menu id

Reply:
Also let's add a test that make sure aria-activedescendant added when user using focus and arrows? (snapshot tests checking component state while it's not focused)